addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -3,60 +3,73 @@
-                        <img t-attf-src="/web/image/res.partner/{{registration.partner_id}}/avatar_128" alt="Registration" class="o_image_64_cover"/>
-                    </div>
-                    <div t-if="registration.status === 'confirmed_registration'" class="alert alert-success text-center" role="alert">
-                        <t t-out="registration.name"/> is successfully registered
+                <div class="col-lg-10 w-100 fs-2">
+                    <div t-if="registrationStatus.value === 'confirmed_registration'" class="alert alert-success text-center" role="alert">
+                        <i class="fa fa-solid fa-check-circle"/><br/>
+                        <span>Successfully registered!</span>
                     </div>
                     <div t-else="" class="alert alert-warning text-center" role="alert">
-                        <t t-if="registration.status === 'need_manual_confirmation'">
-                            <span>This ticket is for another event<br/>
-                            Confirm attendance for <t t-out="registration.name"/>?</span>
+                        <i class="fa fa-solid fa-exclamation-circle"/><br/>
+                        <t t-if="registrationStatus.value === 'need_manual_confirmation'">
+                            <span>This ticket is for another event!<br/>
+                            Confirm attendance ?</span>
                         </t>
-                        <t t-elif="registration.status === 'not_ongoing_event'">
+                        <t t-elif="registrationStatus.value === 'not_ongoing_event'">
                             <span>This ticket is not for an ongoing event</span>
                         </t>
-                        <t t-elif="registration.status === 'canceled_registration'">
+                        <t t-elif="registrationStatus.value === 'canceled_registration'">
                             <span>Cancelled registration</span>
                         </t>
-                        <t t-elif="registration.status == 'already_registered'">
-                            <t t-out="registration.name"/><span> is already registered</span>
+                        <t t-elif="registrationStatus.value == 'already_registered'">
+                            <span>Ticket already scanned!</span>
                         </t>
                     </div>
-                    <div t-if="registration.has_to_pay" class="alert alert-danger text-center" role="alert">
-                        The registration must be paid
-                    </div>
                 </div>
             </div>
-            <div id="registration_information" class="row">
+            <div class="row fs-1">
+                <div class="col-lg-12 d-flex align-items-baseline gap-3">
+                    <t t-if="registration.name" t-out="registration.name"/>
+                    <t t-else="" t-out="'Guest #' + registration.id"/>
```

Comment:
Issue is that "Guest" won't be translated (nor exported for translation). You should go through a t-set and intermediate variable so that content is extracted and translated, like ``<t t-set="guest_label">Guest #</t>`` .